wallet/how-to/use-non-evm-networks/starknet/sign-starknet-data.md

```diff
@@ -0,0 +1,6 @@
+---
+description: Sign Starknet data in MetaMask.
+sidebar_position: 5
+---
+
+# Sign Starknet data
```

Comment:
@khanti42  thanks for all your feedback. Can you please add code to this one as well?